pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -191,6 +194,19 @@ async def run(self):
         _LOGGER.info("Updating ocp-build-data...")
         await self.update_build_data(advisories, jira_issue_key)
 
+        if "advance" in advisories.keys():
+            advisory_info = self.get_advisory_info(advisories["advance"])
+            # Make sure that the advisory is in editable mode
+            if not self.is_advisory_editable(advisory_info):
+                raise VerificationError(f"Advisory {advisory_info['id']} is not editable. "
```

Comment:
I would log an info message here instead of erroring out, and continue with the metadata advisory

Reply:
Added in https://github.com/openshift-eng/art-tools/pull/341/commits/9d25fa3ff9e7438cf8e8c711d49ca8580afeb076